docs/content/server/setup.md

```diff
@@ -19,121 +19,99 @@ We also offer this in video format. You can check it out [here](https://github.c
+You can check if you have Node.js & NPM installed by running the following command:
 
 ```bash
-npm install -g yarn
+node -v
+npm -v
 ```
 
-- Installing Poetry
+Once you have Node.js installed, you can proceed to the next step.
 
-Poetry is a package manager for Python. You can install it by running the following command:
-
-```bash
-pip install poetry
-```
-- Installing Docker and Docker Compose
+### Checking if you have Docker & Docker Compose installed
 
 Docker containerizes applications, while Docker Compose orchestrates multi-container Docker applications.
 
-You can follow the steps here:
-
 If you need assistance installing docker:
 https://docs.docker.com/desktop/
-If you need assistance installing docker compose: 
-https://docs.docker.com/compose/install/
 
-### Installing the dependencies
+Docker-compose is included in Docker Desktop, but if you need assistance installing docker compose: 
+https://docs.docker.com/compose/install/
 
-Once you have installed Yarn and Poetry, you can run the following command to install the dependencies:
+You can check if you have Docker installed by running the following command:
 
 ```bash
-cd autogpt_platform/backend
-cp .env.example .env
-poetry install
+docker -v
+docker-compose -v
 ```
 
-**In another terminal**, run the following command to install the dependencies for the frontend:
+Once you have Docker and Docker Compose installed, you can proceed to the next step.
 
-```bash
-cd autogpt_platform/frontend
-yarn install
-```
+## Running the backend services
 
-Once you have installed the dependencies, you can proceed to the next step.
+To run the backend services, follow these steps:
 
-### Setting up the database
+* Within the repository, clone the submodules and navigate to the `autogpt_platform` directory:
+  ```bash
+   git submodule update --init --recursive
+   cd autogpt_platform
+  ```
+  This command will initialize and update the submodules in the repository. The `supabase` folder will be cloned to the root directory.
 
-In order to setup the database, you need to run the following commands, in the same terminal you ran the `poetry install` command:
+* Copy the `.env.example` file available in the `supabase/docker` directory to `.env` in `autogpt_platform`:
+  ```
+   cp supabase/docker/.env.example .env
+  ```
+  This command will copy the `.env.example` file to `.env` in the `supabase/docker` directory. You can modify the `.env` file to add your own environment variables.
 
-   ```sh
-   docker compose up postgres redis -d
-   poetry run prisma migrate dev
-   ```
-After deploying the migration, to ensure that the database schema is correctly mapped to your codebase, allowing the application to interact with the database properly, you need to generate the Prisma database model:
+* Run the backend services:
+  ```
+   docker compose up -d
+  ```
+  This command will start all the necessary backend services defined in the `docker-compose.combined.yml` file in detached mode.
```

Comment:
This should just be `docker-compose.yml` now

Reply:
Updated